src/archivematicaCommon/lib/xml2obj.py

```diff
@@ -13,6 +13,11 @@ def mets_file(src):
     
     file_uuid = None
     for amd in raw.mets_amdSec:
+        # Some amdSecs are other types of metadata (for instance sourceMD),
+        # and we don't care about those; skip them.
+        if amd.mets_techMD is None:
```

Comment:
It is, yes! `DataNode.__getattr__` returns `None` specifically if the requested attribute isn't present.